streams/src/test/resources/log4j.properties

```diff
@@ -18,4 +18,9 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
-log4j.logger.org.apache.kafka=INFO
+log4j.logger.kafka=WARN
```

Comment:
Do we really want to degrade broker side logs to WARN?

Reply:
We've been discussing this...the way I see it we have three options:
1) the current proposal (demote brokers to WARN)
2) just demote ZK to ERROR and see if that helps enough
3) do a finer grained control of the broker logs, ie demote some less helpful packages to WARN but leave others at INFO

I personally am fine with either 1 or 2 for now, as we can always go back and change it again if it's too little or too much. But I'd rather get fewer broker logs and some Streams logs than no Streams logs at all 🙂

Reply:
Got it, that makes sense.

Are there are particular Streams sub-packages we are interested in, or just `o.a.k.streams.integration` itself is good enough?

Reply:
Hm yeah I guess we should try setting everything to DEBUG. Just trying to find the sweet spot where we can actually get the logs that are useful